packages/shared-types/src/BaseTheme.ts

```diff
@@ -133,13 +133,27 @@ type BaseThemeVariables = {
   typography: Typography
 }
 
+type BaseThemeVariableKeys = [
+  'borders',
+  'breakpoints',
+  'colors',
+  'forms',
+  'media',
+  'shadows',
+  'spacing',
+  'stacking',
+  'transitions',
+  'typography'
+]
```

Comment:
you don't need to type out here, I'd use just 
```
const baseThemeProps = [
  'borders',
...
] as const
```
(unless `BaseThemeVariableKeys` is something that will be used by our users)

Reply:
I didn't want to put a `const` here, since this package contains types only, and couldn't find a better place to put it to... so that's how I solved it :) What do you think?

Reply:
OK :)